src/Agent.Sdk/MountVolume.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Agent.Sdk
+{
+    public class MountVolume
+    {
+        public MountVolume()
```

Comment:
Do we need this?

Reply:
yes, I know we need this one for JSON serialization